packages/shared/src/lib/standardizeUrl.ts

```diff
@@ -0,0 +1,22 @@
+const excludeFromStandardization = ['youtube.com'];
+
+const isExcluded = (url: string) =>
+  excludeFromStandardization.some((e) => url.includes(e));
+
+const subtractDomain = (url: string): string => {
+  const matches = url.match(
+    /^(?:https?:\/\/)?(?:[^@/\n]+@)?(?:www\.)?([^:/?\n]+)/i,
+  );
+  return matches && matches[1];
+};
+
+const removeTrailingSlash = (url: string): string => url.replace(/\/$/, '');
+
+export const standardizeURL = (url: string): string => {
```

Comment:
In the future, we will have to do the standardization on the server-side for sure because this list will be dynamic.
Do we want to fix it before we launch perhaps? But waiting holding back the internal testing

Reply:
Let's fix before launch, but after internal if that makes sense?
I'll simply send the full URL to the API and add this function there.

Let me know if that works for you?

Reply:
@idoshamun Removed from this project and introduced the same on the API level.
https://github.com/dailydotdev/daily-api/pull/946